### PR TITLE
Added coreDNS experiments in sidebars and added new experiments

### LINKED ISSUE #1094: CoreDns experiments should be added in docs
**What happened**:

- CoreDns experiment should be added in litmus docs.

- CoreDns experiment should also be added in sidebars of litmus docs.

- Add all new generic experiment information i.e `pod-network-corruption`,`node-drain`,`pod-cpu-hog`

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by amitbhatt818**
- CoreDns experiment are added in litmus docs.

- CoreDns experiment are also added in sidebar of litmus docs.

-  Added all new generic experiment information i.e `pod-network-corruption` , `node-drain`,`pod-cpu-hog`

Fixes : https://github.com/litmuschaos/litmus/issues/1094
Signed-off-by: Amit Bhatt <amitbhatt818@gmail.com>

### CODE REVIEW

File: docs/drain-node.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ sidebar_label: Drain Node
 
 | Type      | Description                                  | Tested K8s Platform                                               |
 | ----------| -------------------------------------------- | ------------------------------------------------------------------|
-| Generic   | Drain the node where application pod is scheduled. |  GKE, Konvoy(AWS), Packet(Kubeadm), Minikube |
+| Generic   | Drain the node where application pod is scheduled. |  AWS , Packet(Kubeadm) , Konvoy |
```
Comment (ksatchit) [inline docs/drain-node.md]: GKE, Konvoy (AWS), AWS(KOPS), Packet(Kubeadm)

File: docs/pod-network-loss.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ sidebar_label: Pod Network Loss
 
 | Type      | Description              | Tested K8s Platform                                               |
 | ----------| ------------------------ | ------------------------------------------------------------------|
-| Generic   | Inject Packet Loss Into Application Pod | GKE, Packet(Kubeadm), Minikube > v1.6.0 |
+| Generic   | Inject Packet Loss Into Application Pod | GKE, Packet(Kubeadm) , Minikube > v1.6.0 |
```
Comment (ksatchit) [inline docs/pod-network-loss.md]: ?

File: docs/pod-network-latency.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ sidebar_label: Pod Network Latency
 
 | Type      | Description              | Tested K8s Platform                                               |
 | ----------| ------------------------ | ------------------------------------------------------------------|
-| Generic   | Inject Network Latency Into Application Pod | GKE, Packet(Kubeadm), Minikube > v1.6.0        |
+| Generic   | Inject Network Latency Into Application Pod | GKE, Packet(Kubeadm) , Minikube > v1.6.0 |
```
Comment (ksatchit) [inline docs/pod-network-latency.md]: ?

File: website/sidebars.json [hunk 4]
```diff
@@ -19,12 +19,12 @@
           "container-kill",
           "pod-network-latency",
           "pod-network-loss",
-	  "pod-network-corruption",
-	  "pod-cpu-hog",
+	        "pod-network-corruption",
```
Comment (ksatchit) [inline website/sidebars.json]: try viewing file -- indent seems different/erratic now

**Review by chandankumar4 — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**